CollAction/Controllers/ProjectsController.cs

```diff
@@ -329,8 +329,7 @@ public async Task<JsonResult> FindProjects(int? categoryId, int? statusId)
                 default: statusExpression = (p => true); break;
             }
 
-            var projects = await _projectService.FindProjects(
-                Expression.Lambda<Func<Project, bool>>(Expression.AndAlso(projectExpression.Body, Expression.Invoke(statusExpression, projectExpression.Parameters[0])), projectExpression.Parameters[0]));
+            var projects = await _projectService.FindProjects(Expression.Lambda<Func<Project, bool>>(Expression.AndAlso(projectExpression.Body, Expression.Invoke(statusExpression, projectExpression.Parameters[0])), projectExpression.Parameters[0]), limit).ToListAsync();
```

Comment:
Not in scope for this PR, but using this Expression stuff looks really complicated...

Reply:
I do have some ideas on how to refactor... Might put that in this PR.